xml/Accessibility/CAccPropServicesClass.xml

```diff
@@ -22,7 +22,7 @@
     </Interface>
   </Interfaces>
   <Attributes>
-    <Attribute FrameworkAlternate="netframework-4.0;netframework-4.5;netframework-4.5.1;netframework-4.5.2;netframework-4.6;netframework-4.6.1;netframework-4.6.2;netframework-4.7;netframework-4.7.1;netframework-4.7.2;netframework-4.8">
+    <Attribute FrameworkAlternate="">
```

Comment:
I'm seeing some empty FrameworkAlternate values here. By design @joelmartinez?

Reply:
No, that's definitely a bug that needs to be resolved … The way this works is that it adds each framework moniker to the attribute as it processes each serially, and then at the end if all of the frameworks are present, it removes it. Something is going wonky with this process. 

I don't _think_ a GH issue has been created for this

Reply:
OK, I'll create one if I can't find one about this.